workspace/welcome-to-xod/209-slow-down/patch.xodp

```diff
@@ -1,7 +1,21 @@
 {
   "comments": [
     {
-      "content": "# Slow Down\n\nYou often see the \"Loop\" value bound to `UPD` pins.\n\n\"Loop\" (also known as \"Continuously\") means that XOD tries to pulse these nodes as fast as it can. These nodes evaluate themselves on each transaction.\n\nSometimes it's excessive and may lead to slowing the program performance. Especially if you're working with sensors that depend on timing like an ultrasonic range meter.\n\nTo avoid this, you can replace the \"Continuously\" with another source that pulses with the desired interval or only when necessary. For example, a `clock` node.\n\n## Exercise\n\nMake the temperature display less sporadic.\n\n1. Place a `clock` node.\n2. Link it with the `UPD` pin of the `thermometer-tmp36` node.\n3. Place `pulse-on-change` between `Tc` and `UPD` of the text LCD.\n4. Upload the program.\n\nHeat and cool down the thermometer and observe the incremental changes of the data shown.",
+      "content": "Note, that we're using quick-start node `text-lcd-i2c-16x2` in this example. It has an `ACT` pin instead of `UPD` and it updates the text on LCD only when incoming values are changed.\n\nIn other cases, it's much better to pulse only when it necessary to avoid excessive updates of actuator module.\n\nYou can use `pulse-on-change` node for this.",
```

Comment:
This comment is a kludge. The “quickstart node” term is unknown to the reader. The `ACT`/`ACK` pattern will be familiar already once we apply it to LEDs, servos, and so on.

Suggest removing the note.